src/packages.js

```diff
@@ -393,9 +394,13 @@ export default class Packages {
   addPackages(list) {
     if (list instanceof Array) {
       const append = list
-        .map(iter => ({
+        .map((files = [], ...iter) => ({
           type: 'application',
-          files: [],
+          files: files.map(file =>
```

Comment:
Try doing `(iter.files || [])` here and remove the destructure from `.map()` to fix https://github.com/os-js/osjs-client/pull/143#issuecomment-757379309.

Reply:
Clarification: `files: (iter.files || [])` and then remove the destructure from the `.map()` **above** :sweat_smile: